### PR TITLE
White space

### LINKED ISSUE #165: Anthologies: Whitespace cleanup
Clean up whitespace on anthologies views: 

- [x] Make the [anthologies index view](http://cove-staging.herokuapp.com/anthologies) match whitespace on the [documents index](http://cove-staging.herokuapp.com/documents): including table header, and space between nav and table
- [x] Edit actions on the [anthologies index view](http://cove-staging.herokuapp.com/anthologies) match the documents actions column.
- [ ] Put some whitespace around the add to anthology button on the [documents index](http://cove-staging.herokuapp.com/documents), essentially matching the dashboard widgets.
    Screenshot 2020-01-09 16 41 09
- [x] Move add to anthology controls to make select checkbox column narrower and to put the anthology selection menu next to the add to anthology button Screenshot 2020-01-09 16 46 18
- [x] See what you can do to make the search fields look more settled -- whitespace above/below and between, and line up button with field.
Screenshot 2020-01-09 16 49 03

### PR DESCRIPTION
**Pull request by timdunn22**
Closes #165

### CODE REVIEW

**Comment by timdunn22** [comment]
Make the anthologies index view match whitespace on the documents index:
including table header, and space between nav and table
Edit actions on the anthologies index view match the documents actions
column.
Move add to anthology controls to make select checkbox column narrower
and to put the anthology selection menu next to the add to anthology
button
Cleaned up the search buttons on the documents index page
Put some whitespace around the add to anthology button on the documents index, essentially matching the dashboard widgets

File: Gemfile [hunk 1]
```diff
@@ -5,9 +5,10 @@ ruby '2.2.4'
 gem 'rails', '4.2.1'
 gem 'puma'
 
-gem 'pg'
+gem 'pg', '~> 0.18'
```
Comment (jamiefolsom) [inline Gemfile]: Can you comment on the motivation for the version upgrades of pg, paperclip, apartment? Not sure whether this is deployed to staging, but given that we're having trouble on staging, it seems like these upgrades should be in their own PR, rather than rolled into a PR related to whitespace.
